azurerm/internal/services/apimanagement/resource_arm_api_management.go

```diff
@@ -201,6 +202,23 @@ func resourceArmApiManagementService() *schema.Resource {
 				},
 			},
 
+			"protocols": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+
+						"enable_http2": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
```

Comment:
How do you feel about removing `Computed: true` and instead just have`Default: false` so it matches the way the api is implemented? Optional Computed can get a little wonky so we should limit it where we can.

Reply:
Hi @mbfrahry,
thanks for your comment. It's now set to `Default: false`.